package-lock.json

```diff
@@ -8675,7 +8715,7 @@
         "@babel/plugin-proposal-function-bind": "7.7.4",
         "@babel/plugin-proposal-object-rest-spread": "7.7.4",
         "@babel/plugin-syntax-dynamic-import": "7.7.4",
-        "@babel/preset-env": "^7.4.5",
+        "@babel/preset-env": "^7.8.3",
```

Comment:
This should be done by upgrading `carbon-factory`.

Reply:
this in fact is not needed. removed